src/Domain/Block.hpp

```diff
@@ -87,15 +87,43 @@ class Block {
   const domain::CoordinateMapBase<Frame::Grid, Frame::Inertial, VolumeDim>&
   moving_mesh_grid_to_inertial_map() const;
 
+  /// \brief The map going from the last time independent frame to the
+  /// distorted frame. Only used when the coordinate map is
+  /// time-dependent.
```

Comment:
What exactly is the distorted frame? The grid frame and inertial frame have nice descriptions of what they actually are ("last time independent frame" and "frame in which the equations are solved", respectively). Is the distorted frame a frame after a couple specific maps are applied? If so, which ones?

It may not be clear at the moment, so this is ok as is, but moving forward once we concretely define what the distorted frame is, these comments should be updated.

Reply:
Yes, there should be more documentation here about what the distorted frame is.

The distorted frame is the frame that you are in if you apply the (inverse) scaling+rotation+translation maps starting from the Inertial frame.   Equivalently, the distorted frame is the frame that you are in if you apply the (forward) shape+size maps starting at the Grid frame.  [I am ignoring Skew and CutX for now].   In evolutions we find the horizon in the distorted frame, because passing the distorted-frame horizons into the control systems decouples the different control errors (allowing each control system to work independently instead of having some huge coupled system).

Reply:
I don't think here's the right place. We have some central "domain concepts" documentation that describes these sorts of things. There's no way I'd ever think to look at the `Block` documentation to learn what the `Distorted` frame is

Reply:
Agreed.  A description can be added to `docs/DevGuide/DomainConcepts.md` and perhaps a comment here could point there.

Reply:
:+1: I like the suggestion of having a comment here pointing to the DomainConcepts :)